dist/css/bootstrap.css

```diff
@@ -6456,9 +6460,39 @@ tbody.collapse.in {
        -o-animation: showDropdown .2s cubic-bezier(.6, .2, .1, 1) both;
           animation: showDropdown .2s cubic-bezier(.6, .2, .1, 1) both;
 }
+.open > .dropdown-menu-native {
+  display: block;
+  -webkit-animation: slideIn .2s both;
+       -o-animation: slideIn .2s both;
+          animation: slideIn .2s both;
+}
 .open > a {
   outline: 0;
 }
+@-webkit-keyframes slideIn {
+  from {
+    margin-bottom: -100%;
```

Comment:
We need to animate with `translateY`, margin can't be GPU accelerated and causes repaints during animation.